codelabs/Getting_Started_with_gRPC_Java/start_here/src/main/proto/routeguide/route_guide.proto

```diff
@@ -12,13 +12,17 @@ package routeguide;
   /////////////////////////////////////////////////////////////////////////////
 
 
-message Point {
+message Feature {
   /////////////////////////////////////////////////////////////////////////////
-  // Codelab Hint: Define the Point message's contents here
+  // Codelab Hint: Define the Feature message's contents here
   /////////////////////////////////////////////////////////////////////////////
 }
 
   /////////////////////////////////////////////////////////////////////////////
-  // Codelab Hint: Define the Feature message here
+  // Codelab Hint: Define the Point message here
```

Comment:
you removed the `message Point {`

Reply:
I flipped around which message needs to be filled in and which needs to be created from scratch.  I really want them to type the keyword "message" so that it sticks.

Reply:
i see - makes sense then